scanner/mappers/mappers_test.go

```diff
@@ -654,7 +654,7 @@ func Test_toProtoV4VulnerabilitiesMap(t *testing.T) {
 				},
 			},
 		},
-		"when vuln with range and no fixedIn then use upper limit": {
+		"when vuln with range and no fixedIn then not fixed": {
 			ccVulnerabilities: map[string]*claircore.Vulnerability{
 				"foo": {
 					Issued: now,
```

Comment:
You might want to remove the Ranges in the vulns now that we know ClairCore does not populate in the reports.